recipe/activate.sh

```diff
@@ -33,6 +33,7 @@ then
     export CXXFLAGS="${CXXFLAGS} -DBOOST_MATH_DISABLE_FLOAT128"
     export LDFLAGS="${LDFLAGS}"
     export LINKFLAGS="${LDFLAGS}"
+    export MAKEFLAGS="-j2 ${MAKEFLAGS}"
```

Comment:
Would you mind adding a comment to help future maintainers understand the rationale?